--- origsrc/ctpl-0.3.3/configure.ac	2011-08-10 19:53:54.000000000 -0500
+++ src/ctpl-0.3.3/configure.ac	2013-09-24 15:37:43.126509100 -0500
@@ -10,7 +10,7 @@ AC_INIT([CTPL],
 AC_CONFIG_SRCDIR([src/ctpl.h])
 AC_CONFIG_AUX_DIR([build/aux])
 AC_CONFIG_MACRO_DIR([build/m4])
-AM_INIT_AUTOMAKE([1.11.1 -Wall -Werror foreign])
+AM_INIT_AUTOMAKE([1.11.1 -Wall foreign])
 AC_CONFIG_HEADERS([config.h])
 
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
