--- origsrc/ctpl-0.3.2/src/Makefile.am	2010-11-06 18:31:20.000000000 -0500
+++ src/ctpl-0.3.2/src/Makefile.am	2011-01-12 19:37:53.814669400 -0600
@@ -5,7 +5,7 @@ lib_LTLIBRARIES = libctpl.la
 
 libctpl_la_CPPFLAGS = -DG_LOG_DOMAIN=\"CTPL\" -DCTPL_COMPILATION
 libctpl_la_CFLAGS   = @GLIB_CFLAGS@ @GIO_CFLAGS@
-libctpl_la_LDFLAGS  = -version-info @CTPL_LTVERSION@
+libctpl_la_LDFLAGS  = -version-info @CTPL_LTVERSION@ -no-undefined
 libctpl_la_LIBADD   = @GLIB_LIBS@ @GIO_LIBS@ -lm
 libctpl_la_SOURCES  = ctpl-environ.c \
                       ctpl-eval.c \
